src/Microsoft.Azure.AppConfiguration.Functions.Worker/AzureAppConfigurationRefreshMiddleware.cs

```diff
@@ -24,9 +24,15 @@ public AzureAppConfigurationRefreshMiddleware(IConfigurationRefresherProvider re
 
         public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next)
         {
-            foreach (IConfigurationRefresher refresher in Refreshers)
+            //
+            // Configuration refresh is meant to execute as an isolated background task.
+            // To prevent access of request-based resources, such as HttpContext, we suppress the execution context within the refresh operation.
+            using (AsyncFlowControl flowControl = ExecutionContext.SuppressFlow())
             {
-                _ = refresher.TryRefreshAsync();
+                foreach (IConfigurationRefresher refresher in Refreshers)
+                {
+                    _ = Task.Run(() => refresher.TryRefreshAsync());
```

Comment:
Can this be `_ = Task.Run(refresher.TryRefreshAsync);`

Reply:
Did you mean something else? It `cannot convert from 'method group' to 'System.Action'`.